entrypoint.sh

```diff
@@ -155,6 +156,11 @@ try_update_spt() {
     exit 0
 }
 
+spt_listen_on_all_networks() {
+    # Changes the ip and backendIp to 0.0.0.0 so that the server will listen on all network interfaces.
+    sed -i 's/127.0.0.1/0.0.0.0/g' $mounted_dir/SPT_Data/Server/configs/http.json
```

Comment:
A good approach, but this assumes that the IP configured here is `127.0.0.1`, which would work for the default but may not if someone is moving over their SPT server install for example. How about using `jq` to be more explicit? e.g. 
```suggestion
# Haven't tested this, but something like...
json_path=$mounted_dir/SPT_Data/Server/configs/http.json
modified_http_json="$(jq '.ip = "0.0.0.0" | .backendIp = "0.0.0.0"' $json_path)" && echo -E "${modified_http_json}" > $json_path
```

Reply:
You're right, it does assume that. It is always 127.0.0.1 in my case but may not be for everyone so I agree.

Reply:
I like that solution. I did a bit of testing with it and it works. I'll add it to a commit